doc/bc/changes-8.0.md

```diff
@@ -55,6 +55,22 @@ Changes affecting version compatibility with former or future versions.
 * Deprecated `eZ\Publish\SPI\FieldType\EventListener` interface, `eZ\Publish\SPI\FieldType\Event` class and
   `eZ\Publish\SPI\FieldType\Events` namespace have been dropped.
 
+* Clean Installer (`EzSystems\PlatformInstallerBundle\Installer\CleanInstaller`), its service
+  definition `ezplatform.installer.clean_installer`, and the
+  `ezplatform.installer.clean_installer.class` parameter have been dropped. Use Core Installer
+  instead (`EzSystems\PlatformInstallerBundle\Installer\CoreInstaller`) along with
+  `EzSystems\DoctrineSchema\API\Event\SchemaBuilderEvents::BUILD_SCHEMA` event (or just that event).
+  See eZ Platform Documentation for more details.
+
+* The `ezplatform.installer.db_based_installer.class` and `ezplatform.installer.install_command.class`
+  parameters have been dropped. The `ezplatform.installer.db_based_installer` service definition has
+  been dropped as well. Instead, the FQCN-named service
+  `EzSystems\PlatformInstallerBundle\Installer\DbBasedInstaller` is available.
+
+* `EzSystems\PlatformInstallerBundle\Command\InstallPlatformCommand` has been marked as final.
+  Overriding it was never supported. Use SchemaBuilder event-oriented extension point to inject
+  custom behavior into the installation process (see eZ Platform Documentation for more details).
```

Comment:
@ezsystems/documentation-team if you have time, please review the changes in this doc.

Reply:
> @ezsystems/documentation-team if you have time, please review the changes in this doc.

Thanks for the review, applied your suggestions.